pkg/upgrade/upgrade.go

```diff
@@ -273,19 +273,19 @@ func getDashboardWatsonResources(ns string) []ResourceSpec {
 
 	return []ResourceSpec{
 		{
-			Gvk:       gvk.OdhQuickStart,
+			Gvk:       cluster.OdhQuickStart,
```

Comment:
Why? It was put in a separate package on purpose, to have own namespace for GVKs. Otherwise like in the original proposal it may be wrapped by a anonymous structure which makes path longer.

Reply:
>  i dont feel we should split them into two different package.

yes, 100% they should be in one package available all around the code, that was the point.

Reply:
how you feel about this way:
i can have a gvk.go with all 6 defined there, but put the gvk in the cluster package?

Reply:
Fine for me, it's ok to import "cluster/gvk"